src/depcache.erl

```diff
@@ -812,3 +840,6 @@ flush_message(Msg) ->
     after 0 ->
             ok
     end.
+
+callback(Type, Name, {M, F, A}) ->
+    spawn(M, F, [{Type, Name} | A]).
```

Comment:
Just call directly, the callee can still spawn.
In general we will just add a number to some ets entry, so the spawn won't be needed.